cache-worker/index.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * This file is part of Serlo.org API
+ *
+ * Copyright (c) 2020 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2020 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/api.serlo.org for the canonical source repository
+ */
+import { CacheWorker } from './src/cache-worker'
+import { Service } from './src/lib'
+
+const apiEndpoint = process.env.SERLO_ORG_HOST
+const secret = process.env.SERLO_ORG_SECRET
+const service = process.env.SERLO_SERVICE as Service
+const cacheKeys = process.env.CACHE_KEYS
+
+const cw = new CacheWorker({ apiEndpoint, secret, service })
+
+const MAX_RETRIES = (process.env.CACHE_KEYS_RETRIES as unknown) as number
+
+async function start() {
+  console.log('Updating cache values of the following keys:', cacheKeys)
+  run(0)
+}
+
+async function run(tries: number) {
+  await cw.updateCache(cacheKeys!)
```

Comment:
```suggestion
async function run(cw: CacheWorker, retries: number) {
  await cw.updateCache(cacheKeys!)
```

Here you introduced a side effect to the `run()` function (see https://github.com/labs42io/clean-code-typescript#avoid-side-effects-part-1 ). A side effect is an effect a function has besides its return value (in this case changing a global variable). It's is often better to avoid side effects. Code with side effects is harder to test (You cannot just test the return value, you also need to test the side effect itself), it is harder to read (I needed to see where `cw` is defined), and code with side effects is harder to parallelize.